src/s390x/dasd.rs

```diff
@@ -174,19 +177,10 @@ fn is_formatted(dasd: &str) -> Result<bool> {
 /// # Arguments
 /// * `dasd` - dasd device, i.e. smth like /dev/dasda
 fn is_invalid(dasd: &str) -> Result<bool> {
-    let cmd = Command::new("fdasd")
-        // we're looking for a hardcoded string in the output
-        .env("LC_ALL", "C")
-        .arg("-p")
-        .arg(dasd)
-        .output()
-        .chain_err(|| format!("executing fdasd -p {}", dasd))?;
-    if !cmd.status.success() {
-        bail!("fdasd -p {} failed", dasd);
-    }
-    Ok(std::str::from_utf8(&cmd.stdout)
-        .chain_err(|| "decoding fdasd output")?
-        .contains("disk label block is invalid"))
+    let mut cmd = Command::new("fdasd");
+    // we're looking for a hardcoded string in the output
+    cmd.env("LC_ALL", "C").arg("-p").arg(dasd);
```

Comment:
Nit: why not continue to chain the builder calls directly from `Command::new()`?

Reply:
We need to bind the `Command` object to a variable.

Reply:
Whoops, yes, you're right.  `arg()` returns `&mut`.